sympy/printing/str.py

```diff
@@ -311,6 +311,10 @@ def _print_HadamardProduct(self, expr):
         return '.*'.join([self.parenthesize(arg, precedence(expr))
             for arg in expr.args])
 
+    def _print_KroneckerProduct(self, expr):
+        return 'x'.join([self.parenthesize(arg, precedence(expr))
```

Comment:
The str printer should be a recreatable form. Pretty printing should go in the pretty printer.

Reply:
Ah, my apologies. I was using the printing of other classes in the `sympy.matrix.expressions` module as a guideline. I'll remove the method in the str printer.